htdocs/install/mysql/migration/11.0.0-12.0.0.sql

```diff
@@ -352,4 +352,4 @@ INSERT INTO llx_c_shipment_mode (rowid,code,libelle,description,tracking,active)
 INSERT INTO llx_c_shipment_mode (rowid,code,libelle,description,tracking,active) VALUES (13,'DPD', 'DPD', NULL, 'https://www.dpd.fr/trace/{TRACKID}', 0);
 INSERT INTO llx_c_shipment_mode (rowid,code,libelle,description,tracking,active) VALUES (14,'MAINFREIGHT', 'Mainfreight', NULL, 'https://www.mainfreight.com/track?{TRACKID}', 0);
 
-
+ALTER TABLE `llx_socpeople` CHANGE `poste` `poste` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL DEFAULT NULL;
```

Comment:
@atm-lena do not use backquote
`ALTER TABLE llx_socpeople MODIFY COLUMN poste VARCHAR(255);`